### DISCUSSION TITLE
Your Voice Matters: Share Your Feedback and Ideas for Improvement!

### DISCUSSION THREAD

**Discussion by jvJUCA** [ruxailab/RUXAILAB#370] [Announcements]
Dear Cooperators,

Your insights and experiences are invaluable to the improvement of our project. We encourage you to actively participate in enhancing our platform by reporting any potential improvements or bugs you encounter during your usage.

Your feedback plays a crucial role in shaping the future of our project, ensuring its continued growth and stability. By sharing your observations and suggestions, you contribute to a more robust and user-friendly platform for everyone.

Whether you've identified a bug, have an idea for a new feature, or simply have a suggestion for improvement, we want to hear from you. Please don't hesitate to create a new issue detailing your findings. Our team will review and prioritize these issues accordingly, working towards implementing the most impactful changes.

Thank you for your collaboration and dedication to making our project even better.

Best regards,
RUXAILAB Organization

**Comment by pankaj87072** [comment]
hello @jvJUCA as i am analyzing the website performance and speed-test 
![Screenshot from 2024-03-12 21-42-15](https://github.com/ruxailab/RUXAILAB/assets/112722707/996f6d01-b388-4d2e-b436-449345571088)
![Screenshot from 2024-03-12 21-42-21](https://github.com/ruxailab/RUXAILAB/assets/112722707/a921d2b8-2715-4701-8947-44989b91b37c)
![Screenshot from 2024-03-12 21-42-24](https://github.com/ruxailab/RUXAILAB/assets/112722707/a6c7b5d1-f481-4c04-8d73-5c9ba08c72f2)
 i got this,so we can enhance the performance by doing minimal changes in 'css' only so i am suggesting the one's who is working on responsiveness should correct the code and use preloading and lazyloading method for the image loading

**Reply by KarinePistili** [reply]
Hi @pankaj87072 . Nice one! We also find it interesting to improve those conditions. We will create an issue from your suggestion. Would you like to work on it?

**Reply by KarinePistili** [reply]
Create issue #371

**Comment by Raju-Mannem** [comment]
> **_Suggestion_**

Need to update the dependencies versions of **`web-eye-tracker-front`** project. 
Although it is not necessary at this point of time. we may face versioning issues if this continues. The application may run slowly.

![dependencies-update](https://github.com/ruxailab/RUXAILAB/assets/104633105/5ef2a225-40c5-4a13-81d5-d4c183897856)

**Reply by KarinePistili** [reply]
Thank you for your suggestion @Raju-Mannem ! I'll open an issue for this. If you feel like working on this issue, ping us ;)

**Reply by asr2003** [reply]
I think some of the packages have been deprecated and have to replace with equivalent packages that has LTS

**Reply by Raju-Mannem** [reply]
Sure @KarinePistili 
Will work on this issue.
Just need your support and attention.

**Comment by asr2003** [comment]
>Suggestion

@KarinePistili What about adding CI/CD pipeline into our project with help of github actions and workflow. It greatly helps to our project we can use it's power of automation to significantly streamline code development and delivery.

**Reply by KarinePistili** [reply]
That would be nice. Do you have any suggestions for an initial pipeline?

**Reply by asr2003** [reply]
There are many in my mind, will create new issue for each github actions and workflows added to our project but lets first start **Pull Request Checks**

**Reply by asr2003** [reply]
@KarinePistili I have raised an issue and I will make sure  with your constant support, together with fellow contributors can transform our project into a top-notch solution that sets the standard for excellence in the industry.

**Reply by KarinePistili** [reply]
Great!

**Comment by TheShiveshNetwork** [comment]
Hey @jvJUCA I came over this error in the console for the project [RUXAILAB](https://github.com/ruxailab/RUXAILAB) don't know whether this is a issue for everyone or not but do let me know. I am attaching the screenshot here:
![image](https://github.com/ruxailab/RUXAILAB/assets/84664410/49a8170d-9bb0-4051-b56b-be9e1908c723)

**Reply by KarinePistili** [reply]
Thank your for reporting

**Comment by TheShiveshNetwork** [comment]
Hey @KarinePistili currently the home page looks like this: 
![image](https://github.com/ruxailab/RUXAILAB/assets/84664410/08f8a79f-d6c8-4694-9113-c33f13f19a8b)
This needs improvement in UI and some components need to be added.

If this is a proper issue do let me know, so that I can start working on this one.

**Reply by KarinePistili** [reply]
This issue has been fixed in latest commits. Thank you for reporting.

**Comment by gitone912** [comment]
<img width="1680" alt="Screenshot_2024-03-23_at_11 53 09_PM" src="https://github.com/ruxailab/RUXAILAB/assets/96557682/26e69864-a823-4e69-9b25-dad50209b0b4">
<img width="551" alt="Screenshot_2024-03-24_at_9 16 30_AM" src="https://github.com/ruxailab/RUXAILAB/assets/96557682/c12912f0-956f-47c6-b5ec-20272bf49cab">
<img width="551" alt="Screenshot_2024-03-24_at_9 15 55_AM" src="https://github.com/ruxailab/RUXAILAB/assets/96557682/3aa84c57-60f3-4835-a781-9c5becffa9b4">

these three issues i encountered while setting up for eye web tracker , however i solved these but i would suggest to fix these bugs to enhance to ease

**Reply by KarinePistili** [reply]
Thank you for reporting

**Comment by asr2003** [comment]
@KarinePistili @jvJUCA Can you please took a look about it 


![image](https://github.com/ruxailab/RUXAILAB/assets/162500856/b96ef2be-df28-4725-ba7f-2e358ec0eb6c)

**Reply by KarinePistili** [reply]
Hey @asr2003 , try again with this link: https://discord.gg/vr6xVZ3H

**Comment by asr2003** [comment]
@KarinePistili Can you also please clarify the heatmaps tool is to deploy publicly like a chrome extension or we are going to use it internally?

I am busy working on this project proposal  from 3 weeks and researched different strategies that can be effectively used to implement this tool and only left with deployment options.  Will soon send you my draft proposal to get reviewed if i get clarified about deployment options :)

**Reply by asr2003** [reply]
And my inactivity is due to my university examinations. From now I will gonna contribute as we discussed to support CI/CD pipeline to our project as per industry best practices

**Reply by KarinePistili** [reply]
Hello @asr2003 , It would be nice to be a tool that can be publicly acessed by people and used easily without needing much programming knowledge

**Comment by elenamolero** [comment]
Hi, How could I find the discord page? I am really exited to contribute with you!

**Comment by dhruvthakur1** [comment]
hey @jvJUCA , while exploring RUXAILAB website i came across this:
![Screenshot 2025-03-28 085322](https://github.com/user-attachments/assets/b835d2e7-174d-4944-8be3-ee4d4baab376)
- when we go to reports section in any particular test.

![Screenshot 2025-03-27 224816](https://github.com/user-attachments/assets/65efbd6f-718f-459b-bb8e-9328a7edd509)
- Issue Description: The "More" button (vertical three dots) under the Reports section is unresponsive and doesn't open any options when clicked.

- Expected Behavior: Clicking on the "More" button should display additional options related to the report (e.g., View, Edit, Download, etc.).

Steps to Check for yourself:
Go to the Reports page.
Click on the three-dot More button.
Observe that no menu or options appear.


Please look into this issue. Let me know if you need any additional details.

**Comment by Shroffx-n** [comment]
Hey @jvJUCA 
I don't know if this is related to the main Ruxailab repo, but I was interested in the AI-Driven NLP Engine project for this year. But the repo for the project was unavailable(I saw a few repos for other project ideas) or I missed it.... 
Can you provide a clarification for this?
Thanks!

**Reply by rakshityadav1868** [reply]
Yeah there is no repo for the nlp project
